assets/js/admin/form-builder.js

```diff
@@ -1068,6 +1353,29 @@
 			field.closest( '.evf-admin-grid' ).find( '[data-field-id="' + old_key + '"]' ).after( newFieldCloned );
 			$(document).trigger('everest-form-cloned', [ new_key, field_type ]);
 			EVFPanelBuilder.switchToFieldOptionPanel(new_key);//switch to cloned field options
+
+			// Process the cloned data if it is a Range Slider field.
+			if ( 'range-slider' === field_type ) {
+				newFieldCloned.find( '.irs' ).remove();
+				newFieldCloned.find( '.evf-slider .evf-range-slider-preview' ).hide();
+
+				// Purify Handle Color Picker.
+				var html = newOption.find( '.everest-forms-field-option-row-handle_color .wp-picker-input-wrap label' ).html();
+				newOption.find( '.everest-forms-field-option-row-handle_color .wp-picker-container' ).remove();
+				newOption.find( '.everest-forms-field-option-row-handle_color' ).append( html );
+
+				// Purify Highlight Color Picker.
+				var html = newOption.find( '.everest-forms-field-option-row-highlight_color .wp-picker-input-wrap label' ).html();
+				newOption.find( '.everest-forms-field-option-row-highlight_color .wp-picker-container' ).remove();
+				newOption.find( '.everest-forms-field-option-row-highlight_color' ).append( html );
+
+				// Purify Track Color Picker.
+				var html = newOption.find( '.everest-forms-field-option-row-track_color .wp-picker-input-wrap label' ).html();
+				newOption.find( '.everest-forms-field-option-row-track_color .wp-picker-container' ).remove();
+				newOption.find( '.everest-forms-field-option-row-track_color' ).append( html );
```

Comment:
Any reason you're re-initializing `html` here? Please avoid doing so, as you're essentially adding a small overhead to the stack unnecessarily, if you can easily just re-utilize the variable without re-initialization.

Reply:
Resolved with 38e1bd4fdd984eab4133668204c60ca1ca725b73